gui.py

```diff
@@ -31,10 +33,14 @@ class Gui(EventHandler):
 
     def __init__(self):
         self.font = self.__identify_font(platform.system())
-        self.app = App("Lemon-Pi",
+        self.root = App("Lemon-Pi",
```

Comment:
I chose to rename the 'app' component to 'root' because self.app is referenced all over the place and so I was avoiding a larger scope of change.

I now dangle two panels off the root ... one of them the splash screen and the other the main app window.